wrap_caller.go

```diff
@@ -0,0 +1,25 @@
+package errtrace
+
+import "braces.dev/errtrace/internal/pc"
+
+// Caller represents a single caller frame, intended for ...
+type Caller struct {
+	callerPC uintptr
+}
+
+// GetCaller captures the program counter of a caller, primarily intended for
+// error helpers so caller information captures the helper's caller.
+//
+// Note: Callers of this function should be marked using `//go:noinline`
```

Comment:
nit: godoc doesn't do markdown formatting. just single quotes will suffice.
```suggestion
// Note: Callers of this function should be marked using '//go:noinline'
```